airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/avro/JsonSchemaType.java

```diff
@@ -16,8 +16,9 @@
 public enum JsonSchemaType {
 
   STRING("string", true, null, Schema.Type.STRING),
-  NUMBER_INT("number", true, "integer", Schema.Type.INT),
-  NUMBER_LONG("number", true, "big_integer", Schema.Type.LONG),
+  // We currently don't distinguish between int and long
+  NUMBER_INT("number", true, "integer", Schema.Type.LONG),
+  NUMBER_BIGINT("number", true, "big_integer", Schema.Type.STRING),
```

Comment:
no sources actually produce big_integer yet, so we can freely change it from meaning int64 to meaning unbounded in t.

Reply:
actually, it looks like source-zoho-crm does (https://github.com/airbytehq/airbyte/blob/edgao/int_type/airbyte-integrations/connectors/source-zoho-crm/source_zoho_crm/types.py#L149) so that needs to remain supported.